src/app/pro/billing/page.tsx

```
@@ -0,0 +1,226 @@
+              <Separator />
+              <div className="flex items-center justify-between text-sm">
+                <span className="text-muted-foreground">Purchase Date</span>
+                <span>{new Date(purchaseDate).toLocaleDateString()}</span>
+              </div>
+            </>
+          )}
+
+          <Separator />
+
+          <div className="space-y-2">
+            <h4 className="font-medium">What's included:</h4>
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)